### PR TITLE
Elasticsearch configurable index

### PR DESCRIPTION
**Pull request by awillis**
Allow for a unified elasticsearch index to be configured. Also includes minor fix to comment and .gitignore.

### CODE REVIEW

File: connector/elasticsearch/elasticsearch/elasticsearch.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def save(self, payload, **kwargs):
         """
         Save results to elasticsearch
 
-        :param bytes payload: Content to be inserted into elasticsearch
+        :param str payload: JSON document to be inserted into elasticsearch
```
Comment (mlaferrera) [inline connector/elasticsearch/elasticsearch/elasticsearch.py]: This is actually going to be a `dict()` object for ES

**Comment by mlaferrera** [comment]
I made a few changes for simplicity sake. Are you ok with them as they stand?

**Comment by awillis** [comment]
So es_index_name will be either a bool or a string?

**Comment by mlaferrera** [comment]
Yep, if False, it will just be skipped. Otherwise the value will be a string and that will be the index name.

**Comment by awillis** [comment]
:+1:  Not my first preference, but it should work all the same

**Comment by akusei** [comment]
@mlaferrera But I want to name my index _False_, I can't do that with this solution. Having 2 separate config values means I can configure a single index with any name I want (including naming it False) and turn on or off the single index functionality without having to change my index name. I feel that having this in 1 variable causes confusion and muddles config data

**Comment by mlaferrera** [comment]
@akusei Thanks for the feedback.

Setting an index name to `False` will not work no matter what as the framework will see it as a `bool` and set it as such. Not to mention, naming an ES index to `False` is probably not an ideal naming convention to begin with.

I'm still not a fan of adding an additional configuration setting that doesn't necessarily need to be there. What I did do however was define `es_index_name` upon plugin instantiation and set it to `False` within the plugin itself. This will allow the end user to simply comment out the `es_index_name` configuration line in the file (thus not having to change the value). This should eliminate any confusion/muddling of config data.

**Comment by akusei** [comment]
@mlaferrera A much appreciated middle ground, thank you!

**Pull request merged**